modules/vaos/app/helpers/vaos/appointments_helper.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
```

Comment:
These files were in the controller before. I chose to leave them here rather than move them to the service for a few reasons. One is that these additions put the service over the LoC limit and caused linting errors. The other is that I expect we're going to create some sort of proxy class soon that will live between the controller and the service. These methods will probably end up there. Additionally, these methods were unit tested using `send` because they were private. Having them as helpers allows them to be public, which avoids the awkwardness of `send`.

Reply:
Sounds good, I think this all lines up with our team's expectation of 1.) constructing proxy services down the road and 2.) cleaning up the patterns we use for our specs. This is looking good to me